catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -411,8 +411,15 @@ public void run() {
                 }
 
                 if (startDate != null) {
-                    if (startDate.getTime() > lastExecutionTime) {
-                        shouldRun = false;
+                    long now = System.currentTimeMillis();
+                    if (startDate.getTime() > now) {
```

Comment:
That variable was no longer necessary and I've moved the last remaining check inline.